status/status_test.go

```diff
@@ -119,6 +119,24 @@ func TestFromErrorOK(t *testing.T) {
 	}
 }
 
+func TestConvertKnownError(t *testing.T) {
+	code, message := codes.Internal, "test description"
+	err := Error(code, message)
+	s := Convert(err)
+	if s.Code() != code || s.Message() != message || s.Err() == nil {
+		t.Fatalf("Convert(%v) = %v; want <Code()=%s, Message()=%q, Err()!=nil>, true", err, s, code, message)
+	}
+}
+
+func TestConvertUnknownError(t *testing.T) {
+	code, message := codes.Code(0), "unknown error"
```

Comment:
Likewise regarding using an integer for a code: because this is `Code(0)`, we can't see that this is `OK` instead of `Unknown`, and `Unknown` is the appropriate result.

Reply:
Yes, I understand now.